lib/Registrator.js

```diff
@@ -44,10 +44,14 @@ module.exports = class Registrator
     // Custom Contact header params for REGISTER and un-REGISTER.
     this._extraContactParams = '';
 
-    if (reg_id)
+    // Contents of the sip.instance Contact header parameter.
+    this._sipInstance = '';
+
+    if (this._reg_id)
```

Comment:
Let's remove this conditional, since we always setting it on the top of the constructor. (I know this was already this way).

Reply:
Agreed, I was just leaving it in there because it was already there before.